### ISSUE TITLE
arKitPoint method return value

### ISSUE THREAD

**Issue by kasperekt** [Estimote/iOS-Indoor-SDK#24]
I think there is a problem with your `arKitPoint(fromIndoorLocationPoint:)` method in **3.0.0.alpha.1** version. Documentation says that it returns position in ARKit coordinate system, but when I print it using this method, and I haven't move, it says I'm e.g. at `(3.470437, -6.539561)` point. Shouldn't it show something near `(0, 0)` since ARKit origin will be in the place I'm right at that moment?

This is how my code snippet looks like:
```swift
func indoorLocationManager(_ manager: EILIndoorLocationManager,
                               didUpdatePosition position: EILOrientedPoint,
                               with positionAccuracy: EILPositionAccuracy,
                               in location: EILLocation) {
    print(manager.arKitPoint(fromIndoorLocationPoint: position))
}
```

**Comment by icappsCedric** [comment]
@kasperekt did you ever get any response on this? Or did you manage to find a solution?

**Comment by kasperekt** [comment]
@iCappsCedric no, I didn't. I stopped using beacons at all and found another solution for my problem.
